src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    private Person person; 
+    private Tag tag; 
+    private String operation; 
+    
+    public Tagging(Person p, Tag t, String op) {
```

Comment:
Try to avoid one/two letter abbreviations unless it is a well known usage e.g. using `i` as iteration counter.